ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift

```diff
@@ -78,6 +78,11 @@ class SettingsViewController: TableViewController {
   private let windowProtection: WindowProtection?
   private let ipfsAPI: IpfsAPI
   private let altIconsModel = AltIconsModel()
+  private var prefsChangeRegistrar: PrefChangeRegistrar!
```

Comment:
Shouldn't be implicitly unwrapped, then move the `prefsChangeRegistrar = PrefChangeRegistrar(prefService: braveCore.profile.prefs)` to the init